src/foam/u2/view/RichChoiceView.js

```diff
@@ -207,6 +207,7 @@ foam.CLASS({
 
     ^chevron::before {
       content: '▾';
+      font-size: 1.5em !important;
```

Comment:
Do we need the important?

Reply:
Not necessarily, there's some places that override this though like SMEStyle and SectionDetailPropertyView, I'm wondering if I should just remove them

Reply:
There should be a way to override it without it affecting the main view. But I don't see why we would need to override the chevron size

Reply:
Ok, I see what you mean, the override is not affecting the richchoiceview, but affecting the children of the nanopay controller which my view fallss under
so will remove `!important`, thanks

Reply:
```suggestion
      font-size: 1.5em;
```